deploy/catalog_resources/community/7.8/kiecloud-operator.7.8.0.clusterserviceversion.yaml

```diff
@@ -218,7 +218,7 @@ spec:
                 - name: POSTGRESQL_PROXY_IMAGE_7.7.0
                   value: registry.redhat.io/rhscl/postgresql-10-rhel7:latest
                 - name: DATAGRID_IMAGE_7.7.0
-                  value: registry.redhat.io/jboss-datagrid-7/datagrid73-openshift:1.3
+                  value: registry.redhat.io/jboss-datagrid-7/datagrid73-openshift:1.5
```

Comment:
code should be modified in constants.go to ensure this 7.7.0 image retains the `1.3` image tag.